modules/ATOSBase/src/ATOSbase.cpp

```diff
@@ -84,11 +84,10 @@ void ATOSBase::onRequestObjectIDs(
 	const std::shared_ptr<atos_interfaces::srv::GetObjectIds::Request> req,
 	std::shared_ptr<atos_interfaces::srv::GetObjectIds::Response> res)
 {
-	char path[PATH_MAX];
 	std::vector<std::invalid_argument> errors;
 	RCLCPP_INFO(get_logger(), "Received object ID information request");
 
-	UtilGetObjectDirectoryPath(path, sizeof (path));
+	auto path = Util::getDirectoryPath("objects");
```

Comment:
I wanted to avoid having about 10 different functions that does essentially the same thing, like we had in `util.c`. But a better way to do this is maybe to use `enum` like Victor said in another comment, instead of the way I did it now.